modules/cnf-du-updating-crio.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-updating-crio_{context}"]
+
+= Updating CRI-O
+
+In support of this feature, CRI-O supports new configuration settings. These are applied by the MachineConfig specified above.
+
+----
+[crio.runtime.workloads.{workload-type}]
+  activation_annotation = "target.workload.openshift.io/{workload-type}"
+  annotation_prefix = "resources.workload.openshift.io"
+  resources = { "cpushares" = 0, "cpuset" = "0-1,52-53" }
+----
+
+* Use the `activation_annotation` field to match pods that should be treated as having the workload type. The annotation key on the pod is compared for an exact match against the value specified in the configuration file. In this release, the only supported workload-type is `management`.
+
+* The `annotation_prefix` is the start of the annotation key that passes settings from the admission hook down to CRI-O.
+
+* The `resources` map associates annotation suffixes with default values. CRI-O defines a well-known set of resources and other values are not allowed. The cpuset value must match the kubelet configuration file and the reserved CPU set in the applied PerformanceProfile.
+
+In the management workload case, it is configured as follows:
+----
+[crio.runtime.workloads.management]
+  activation_annotation = "target.workload.openshift.io/management"
+  annotation_prefix = "resources.workload.openshift.io"
+  resources = { "cpushares" = 0, "cpuset" = "0-1,52-53" }
+----
+
+Pods that have the `target.workload.openshift.io/management` annotation will have their cpuset configured to the value from the appropriate workload configuration. The CPU shares for each container in the pod are configured according to the `management.workload.openshift.io/cores` resource limit, which ensures the pods CPU shares are enforced in the same way as would be done for the original cpu.requests resource.
```

Comment:
First thought: "...ensures the pods CPU shares..." sounds possessive, so maybe "CPU shares for the pod..."

Second thought: The end of that last sentence includes "original" and that makes me think the pod mutation is related, but pod mutation isn't mentioned in this file--seems like a transgression against mod docs guidelines.

Consider just removing "in the same..." to the end.